lib/GoogleSpreadsheetWorksheet.js

```diff
@@ -673,9 +673,35 @@ class GoogleSpreadsheetWorksheet {
     // https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/request#DeleteConditionalFormatRuleRequest
   }
 
-  async sortRange() {
-    // Request type = `sortRange`
-    // https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/request#SortRangeRequest
+  // sortSpec is of the format
+  // [[rowIndex, order], [rowIndex, order]...] where order is 'desc' or 'asc'
+  // https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/request#SortRangeRequest
+  async sortRange(sortSpecs, range) {
+    if (!sortSpecs.length) return null;
+    if (!_.isArray(sortSpecs[0])) { sortSpecs = [sortSpecs]; }
+
+    if (!this.headerValues) await this.loadHeaderRow();
+
+    // Allow a few values for order
+    const dirmap = {
+      desc: 'DESCENDING',
+      asc: 'ASCENDING',
+      DESCENDING: 'DESCENDING',
+      ASCENDING: 'ASCENDING',
+    };
+
+    sortSpecs = sortSpecs.map(([idx, dir]) => ({
+      sortOrder: dirmap[dir],
```

Comment:
maybe good to use `toUpperCase` or `toLowerCase` on what gets passed in to support 'desc' and 'DESC' etc